puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/infra/updateJobs.groovy

```diff
@@ -0,0 +1,54 @@
+pipeline {
+    agent { label 'fast' }
+
+    stages {
+        stage('Setup workspace') {
+            steps {
+                checkout([
+                    $class : 'GitSCM',
+                    branches : [[name: 'master']],
+                    extensions: [[$class: 'CleanCheckout']],
+                    userRemoteConfigs: [
+                        [url: 'https://github.com/theforeman/foreman-infra.git']
+                    ]
+                ])
+
+                dir("./ci") {
+                    sh """
+                    virtualenv update_jobs
+                    source update_jobs/bin/activate
+                    pip install jenkins-job-builder
+                    deactivate
+                    """
+                }
+              }
+        }
+
+        stage('Update ci.centos.org jobs') {
+            steps {
+                dir("./ci") {
+                    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'jenkins-centos', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
+                        sh """
+                        source update_jobs/bin/activate
+                        mv ./centos.org/centos_jenkins.ini.example ./centos.org/jenkins_jobs.ini
```

Comment:
If you're going to stick with `.example`, I think this is also setup code and not actual updating.